sdk/log/batch.go

```diff
@@ -148,6 +150,10 @@ func (b *BatchProcessor) poll(interval time.Duration) (done chan struct{}) {
 				return
 			}
 
+			if d := b.q.Dropped(); d > 0 {
+				global.Warn("dropped log records", "dropped", d)
```

Comment:
If we're dropping records, it likely means the queue is overwhelmed. Won't we be making things (slightly) worse by logging even more?
And won't the logged record also be dropped anyway?

Should we record a metric?